server/app/services/settings/AbstractSettingsManifest.java

```diff
@@ -4,21 +4,67 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSortedMap;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigException;
 import java.util.Locale;
 import java.util.Optional;
 import java.util.function.Function;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import play.mvc.Http;
 
 /** Provides behavior for {@link SettingsManifest}. */
 public abstract class AbstractSettingsManifest {
+  private static final String FEATURE_FLAG_SETTING_SECTION_NAME = "Feature Flags";
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AbstractSettingsManifest.class);
 
   private final Config config;
 
   public AbstractSettingsManifest(Config config) {
     this.config = checkNotNull(config);
   }
 
+  /**
+   * Returns a map containing the names of the settings in the "Feature Flags" section mapped to
+   * their current values.
+   */
+  public ImmutableSortedMap<String, Boolean> getAllFeatureFlagsSorted(Http.Request request) {
+    ImmutableSortedMap.Builder<String, Boolean> map = ImmutableSortedMap.naturalOrder();
+
+    for (SettingDescription settingDescription : getAllFeatureFlags()) {
+      String name = settingDescription.variableName();
+      map.put(name, getBool(name, request));
+    }
+
+    return map.build();
+  }
+
+  private ImmutableList<SettingDescription> getAllFeatureFlags() {
```

Comment:
suggestion (non-blocking): rename this function `getAllFeatureFlagsSettingDescriptions` to make it more clear what the function is

Reply:
Done